packages/advisor-components/src/RuleDetails/RuleDetails.scss

```diff
@@ -0,0 +1,91 @@
+@import '~@patternfly/patternfly/sass-utilities/_all';
+
+hr {
+  border: 0.01rem solid #ccc;
+  margin: var(--pf-global--spacer--md) 0 var(--pf-global--spacer--md) 0;
+}
+
+.ins-c-rule-details__split {
+  display: inline-flex;
+  justify-content: space-around;
+  .pf-l-split__item {
+    position: relative;
+  }
+}
+
+.ins-c-rule-details__stack {
+  width: 600px;
```

Comment:
Couldn't this be achieved by `grid` system? I am a bit worried about small/large screens.

Reply:
Yeah, sure it can... the thing is, I was copy-pasting these CSS stylesheets from the previous old implementation and didn't dig into it yet... I'll revisit this part soon and push a commit

Reply:
Rewrote it, however, not by using a grid, but flex instead (this is for me a bit familiar, and the same result is achievable). 
There are still some bits to be tweaked (Actions button position in small devices), however, I would like to tackle it in a separate PR if you don't mind
 
![image](https://user-images.githubusercontent.com/31385370/162191849-037e5e0b-ccef-4203-8213-7e145c2a8e97.png)
![image](https://user-images.githubusercontent.com/31385370/162191886-23e2cc36-e4b6-4edb-8ac6-f19f58161afc.png)